hugo/layouts/index.html

```diff
@@ -27,9 +27,7 @@
         {{ partial "home/blog.html" . }}
     {{ end }}
 
-    {{ partial "footer/slack.html" . }}
-    {{ partial "footer/contact-us.html" . }}
-    {{ partial "footer/sitemap.html" . }}
+    {{ partial "footer.html" . }}
```

Comment:
The slack part is not exactly part of the same block, it just happens to be on top. The email, yes, it is.